code/modules/mob/living/simple_animal/friendly/rabbit.dm

```diff
@@ -0,0 +1,77 @@
+//Good ol' rabbits
+
+/mob/living/simple_animal/rabbit
+	name = "\improper rabbit"
+	desc = "The hippiest hop around."
+	gender = PLURAL
+	mob_biotypes = MOB_ORGANIC|MOB_BEAST
+	health = 15
+	maxHealth = 15
+	icon = 'icons/mob/rabbit.dmi'
+	icon_state = "rabbit_white"
+	icon_living = "rabbit_white"
+	icon_dead = "rabbit_white_dead"
+	speak_emote = list("sniffles","twitches")
+	emote_hear = list("hops.")
+	emote_see = list("hops around","bounces up and down")
+	butcher_results = list(/obj/item/food/meat/slab = 1)
+	can_be_held = TRUE
+	density = FALSE
+	speak_chance = 2
+	turns_per_move = 3
+	response_help_continuous = "pets"
+	response_help_simple = "pet"
+	response_disarm_continuous = "gently pushes aside"
+	response_disarm_simple = "gently push aside"
+	response_harm_continuous = "kicks"
+	response_harm_simple = "kick"
+	attack_verb_continuous = "kicks"
+	attack_verb_simple = "kick"
+	pass_flags = PASSTABLE | PASSMOB
+	mob_size = MOB_SIZE_SMALL
+	gold_core_spawnable = FRIENDLY_SPAWN
+	var/icon_prefix = "rabbit" //passed to animal_varity as the prefix icon.
+
+/mob/living/simple_animal/rabbit/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/pet_bonus, "hops around happily!")
+	AddElement(/datum/element/animal_variety, icon_prefix, pick("brown","black","white"), TRUE)
+
+/mob/living/simple_animal/rabbit/easter
+	icon_state = "e_rabbit_white"
+	icon_living = "e_rabbit_white"
+	icon_dead = "e_rabbit_white_dead"
+	icon_prefix = "e_rabbit"
+	speak = list(
+		"Hop into Easter!",
+		"Come get your eggs!",
+		"Prizes for everyone!",
+	)
+	icon_prefix = "e_rabbit"
+	var/initial_egg_amount = 10 ///passed to egg_layer component as how many eggs it starts out as able to lay.
+
+/mob/living/simple_animal/rabbit/easter/space
+	icon_state = "s_rabbit_white"
+	icon_living = "s_rabbit_white"
+	icon_dead = "s_rabbit_white_dead"
+	icon_prefix = "s_rabbit"
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	maxbodytemp = 1500
+	unsuitable_atmos_damage = 0
+
+/mob/living/simple_animal/rabbit/easter/Initialize(mapload)
+	. = ..()
+	var/eggs_added_from_eating = rand(1, 4)
+	var/max_eggs_held = 8
```

Comment:
May as well move this to `/mob/living/simple_animal/rabbit/easter`. Also seems a bit odd that they start out with more eggs than their max, but i know that's just how they already were and it doesn't cause any code issues currently. Your call if you wanna change this or not

Reply:
Maybe it's so they can lay more eggs (as a surge) when the event starts and then have a lower cap? I'm not interested in rebalancing the easter code in this PR and balancing it, just wanted to move the code around to get rabbits out of easter.dm.

I moved them to the parent.